docs/data/toolpad/core/introduction/tutorial.md

```diff
@@ -1,3 +1,149 @@
-# Tutorial
+---
+title: Tutorial
+---
+
+# Toolpad Core - Tutorial
 
 <p class="description">Tutorial</p>
+
+## Steps to follow
+
+1. Run:
+
+   ```bash
+   yarn create toolpad-app --tutorial my-first-project
```

Comment:
Does it make sense to introduce a new flag for this? Perhaps we can add a tutorial example?
```suggestion
   yarn create toolpad-app --core --example tutorial
```